wp-content/themes/ngisweden/functions/admin_dashboard.php

```diff
@@ -171,7 +197,7 @@
 ?>
 
   <?php if(count($methods_missing_application) > 0): ?>
-    <h3>Methods missing an application</h3>
+    <h3 style="background-color: #ffb3b3">Methods missing an application</h3>
```

Comment:
I could remove it, it was just to differentiate between the pages working and not working, but since the pages working are now at the bottom, I could remove the color :-)